modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -650,6 +650,14 @@ class CV_EXPORTS_W VideoCapture
     */
     CV_WRAP virtual bool open(int index);
 
+    /** @brief  Open a camera for video capturing
+
+    @overload
+
+    Parameters of the methods are open(int cameraNum, int apiPreference),@return open(cameraNum + apiPreference).
```

Comment:
Please place `@return` on a separate line and remove comma before that, because it is visible in [documentation](http://pullrequest.opencv.org/buildbot/export/pr/7625/docs/d8/dfe/classcv_1_1VideoCapture.html#a128a4457358fb7dccb9a9a686e56e344).